scripts/Properties.js

```diff
@@ -0,0 +1,77 @@
+debugger
+var rootUrl = "", siteIdentifier = "", email = "", environment = "", EmbedSecret = "";
+var baseUrl = "";
+var dashboardServerApiUrl = "";
+var dataServiceUrl = "";
+var designerServiceUrl = "";
+var designerRootUrl = "";
+var intermediateDbStatus = false;
```

Comment:
Is this variable using? Cross verify other variables too.

Reply:
Removed unwanted variables